alerts/notifications.py

```diff
@@ -0,0 +1,36 @@
+from django.core.mail import send_mail
+from users.models import User
+from series.models import APIShow
+
+
+class NotificationBackend:
+    """Backend for generic notifications."""
+
+    def notify(self, user: User, tv_show: APIShow):
+        pass
+
+
+class EmailNotifier(NotificationBackend):
+    """Notifications with emails."""
+    _subject = 'A new episode arrives today !'
+    _basic_content = 'Today, a new episode of : '
+    _mail_from = 'new_series@uptv.com'
+
+    def notify(self, user: User, tv_show: APIShow):
+        send_mail(
+            self._subject,
+            self.get_full_content(user, tv_show),
```

Comment:
FYI, Django's templating system could also be used to great extent here. See [this StackOverflow answer](https://stackoverflow.com/a/2810245) as an example. I think we could add this in the future. :)

Reply:
To be done later